agent/xds/listeners.go

```diff
@@ -1022,6 +1023,21 @@ func (s *ResourceGenerator) injectConnectTLSForPublicListener(cfgSnap *proxycfg.
 	return nil
 }
 
+func getAlpnProtocols(protocol string) []string {
+	var alpnProtocols []string
+
+	switch protocol {
+	case "grpc":
+		alpnProtocols = append(alpnProtocols, "h2", "http/1.1")
+	case "http2":
```

Comment:
I think the correct list of protocols is "h2" and "http/1.1" here. Sorry for the mixup.

Reply:
OK. Thanks! I pushed a new commit that switches this.